components/FullPageTabs/FullPageTabs.vue

```diff
@@ -17,15 +20,18 @@
             <span aria-hidden="true">(</span>{{ link.count }}<span aria-hidden="true">)</span>
           </sup>
         </CdataLink>
-      </div>
+      </component>
     </div>
   </div>
 </template>
 
 <script setup lang="ts">
-defineProps<{
+withDefaults(defineProps<{
   links: Array<{ href: string, label: string, count?: number }>
-}>()
+  as?: string
+}>(), {
+  as: 'h2',
```

Comment:
Should it be a `nav`? `h2` with a list of links inside it seems weird…

Reply:
h2 is for the children not the list, but maybe we can name it `tab-as` or something to improve the readability of the props ?

Reply:
Hum… I see `<component :is="as"` englobing the `<CdataLink v-for…`, no?